.github/quarkus-github-bot.yml

```diff
@@ -285,13 +285,17 @@ triage:
         - extensions/resteasy-classic/resteasy-qute/
     - id: reactive-messaging
       labels: [area/reactive-messaging, area/smallrye]
-      title: "reactive.messaging"
+      expression: |
+        matches("reactive.messaging", title)
+        || matches("Mqtt", titleBody)
+        || matches("RabbitMQ", titleBody)
```

Comment:
Let's limit it to `title`. Also let's put the strings in lowercase, it's less confusing.